staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -136,7 +136,7 @@ func (flags *WaitFlags) AddFlags(cmd *cobra.Command) {
 	flags.ResourceBuilderFlags.AddFlags(cmd.Flags())
 
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", flags.Timeout, "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week.")
-	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name]. The default status value of condition-name is true, you can set false with condition=condition-name=false")
+	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name[=condition-value]]. The default condition-value is true.  Condition values are compared after Unicode simple case folding.")
```

Comment:
I dunno, is this actually helping normal humans? I have no idea what “Unicode simple case folding” is. Saying “case insensitive”, while perhaps not precisely accurate, would be more widely understood. WDYT?

Reply:
[Go's `EqualFold` docs][1] say:

> ...Unicode case-folding, which is a more general form of case-insensitivity...

I'm fine adding additional words like that, if folks think they would be helpful here.

[1]: https://pkg.go.dev/strings#EqualFold

Reply:
+1.

Reply:
I've pushed 44a79753886 -> 8a129792565, rebasing on `master` and adding `which is a more general form of case-insensitivity`.